encord/client.py

```diff
@@ -1245,5 +1245,16 @@ def __set_project_ontology(self, ontology: Ontology) -> bool:
         payload = {"editor": ontology.to_dict()}
         return self._querier.basic_setter(OrmProject, uid=None, payload=payload)
 
+    def workflow_reopen(self, label_hashes: list[str]) -> None:
+        """
+        This function is documented in :meth:`encord.objects.LabelRowV2.workflow_reopen`.
+        """
+
+        # Workaround to make basic_setter generate proper api call
+        class LabelWorkflowGraphNode:
+            pass
+
+        self._querier.basic_setter(LabelWorkflowGraphNode, label_hashes, payload=None)
```

Comment:
Thinking about the public API, a "post" to the Label workflow graph node "endpoint" does not indicate well that we're reopening a task. Something to think about when it comes to naming. I'm also not sure how I feel about `workflow_reopen`, but I can't come up with something better for now so probably that's the best candidate.